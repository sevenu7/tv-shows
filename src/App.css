h1 {
  text-align: center;
  margin-top: 20px;
  margin-bottom: 20px;
}

h2 {
  margin-top: 40px;
}

.header {
  background-color: #343a40;
  color: aliceblue;
}

.desc{
  text-align: center;
}

.middle {
  max-width: 350px;
  margin: auto;
}

.searchField {
  border: 1px solid #343a40;
  width: 100%;
  padding: 10px 10px 10px 40px;
  background: url('./assets/searchIcon.png') no-repeat left center;
}

.showCard {
  border: 1px solid black;
}

.infoCard {
  margin-top: 90px;
  margin-bottom: 40px;
  padding: 20px;
  box-shadow: 0px 0px 25px rgba(0, 0, 0, 0.16);
}

.infoImage {
  float: left;
  margin-right: 10px;
}

.thumbnail {
  margin-bottom: 10px;
}

.thumbnail a:hover{
  text-decoration: none;
}

.thumbnail:hover {
  box-shadow: 0px 0px 25px rgba(0, 0, 0, 0.8);
}

.thumbnail-desc {
  text-align: center;
  color: black;
}

.thumbnail-desc:hover {
  text-decoration: none;
}

.spinner {
  position: absolute;
  top: 50%;
  left: 50%;
}