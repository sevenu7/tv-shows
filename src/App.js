import React, { Component } from 'react';
import { Route, Switch } from 'react-router-dom';

import './App.css';
import Dashboard from './Container/Dashboard/index';
import GenreShows from './Container/GenreShows';
import ShowInfo from './Container/ShowInfo';

class App extends Component {

  render() {
    return (
      < Switch >
        <Route path="/genre/:id" exact component={GenreShows} />
        <Route path="/show/:id" exact component={ShowInfo} />
        <Route path="/" exact component={Dashboard} />
      </Switch >
    );
  }

}

export default App;
