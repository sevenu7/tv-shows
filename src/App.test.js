import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import App from './App';

configure({ adapter: new Adapter() });

describe('renders learn react link', () => {
  it("should match Snapshot", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
