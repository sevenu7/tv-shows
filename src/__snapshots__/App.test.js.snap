// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link should match Snapshot 1`] = `
"<Switch>
  <Route path=\\"/genre/:id\\" exact={true} component={[function]} />
  <Route path=\\"/show/:id\\" exact={true} component={[function]} />
  <Route path=\\"/\\" exact={true} component={[function]} />
</Switch>"
`;
